shell/platform/darwin/graphics/BUILD.gn

```diff
@@ -7,19 +7,23 @@ assert(is_ios || is_mac)
 import("//flutter/common/config.gni")
 
 source_set("graphics") {
-  cflags_objc = flutter_cflags_objc
-  cflags_objcc = flutter_cflags_objcc
+  cflags_objcc = [ "-fobjc-arc" ]
```

Comment:
Shouldn't this be in addition to the `flutter_cflags_objc`? I am not sure what those flags currently are but we are no longer applying them to this target.

Reply:
I noticed that the other targets that enable arc, only had this will verify.

Reply:
I created a new high level param for arc targets and unified it.